reader/readerUI.css

```diff
@@ -2,6 +2,7 @@ body, html {
 	height: auto;
 	overflow: auto;
 	min-height: 100vh;
+	background-color: transparent;
```

Comment:
Why is this necessary?

Reply:
I already made a correction, believe it solved it, so it is no longer necessary `background-color: transparent;`